### DISCUSSION TITLE
Subreddit showing description,creation date and member count

### DISCUSSION THREAD

**Discussion by Edfgdz** [Docile-Alligator/Infinity-For-Reddit#1034] [Ideas]
Is it possible to let subreddit show more information with subreddit description,creation date and how many members it has in subreddit search results like this app?

![Screenshot_2021-09-19-12-17-32-781_com rubenmayayo reddit](https://user-images.githubusercontent.com/26460124/133915602-c72a1d7e-4bfe-47d7-9194-78e650428f09.jpg)

At present Infinity can't show them.

![Screenshot_2021-09-19-12-17-06-964_ml docilealligator infinityforreddit](https://user-images.githubusercontent.com/26460124/133915607-05563a7f-2351-4d62-a1a0-700472582d7a.jpg)

Because sometimes there are similar subreddits in search results so if those subreddits could show more information then user could know which subreddit is what they want. 

For example according to subreddit description user would know second one is better for them to choose.

**Comment by Docile-Alligator** [comment]
Good idea.

**Comment by Wladefant** [comment]
Subreddit subscriber count is implemented in [v5.3.0-beta1](https://github.com/Docile-Alligator/Infinity-For-Reddit/releases/tag/v5.3.0-beta1)
